src/sanitizer.js

```diff
@@ -116,6 +115,37 @@ const BLACKLISTED_ATTR_VALUES = [
 ];
 
 
+/** @const {!Object<string, !Object<string, !RegExp>>} */
+const BLACKLISTED_TAG_SPECIFIC_ATTR_VALUES = {
+  'input': {
+    'type': /(?:image|file|password|button)/i,
```

Comment:
Please follow validator here then. But also please send an issue to validator folk about this.